bin/stylus

```diff
@@ -401,19 +401,19 @@ function help(name) {
       url = 'https://developer.apple.com/library/safari/search/?q=' + name;
       break;
     case 'opera':
-      url = 'http://dev.opera.com/search/?term=' + name;
+      url = 'https://dev.opera.com/search/?term=' + name;
```

Comment:
Can you help us update all search pattern ?

```diff
# For example
- https://dev.opera.com/search/?term=
+ https://dev.opera.com/search/?q=
```

Reply:
Yes, I can take a look at that 👍